core-ui/src/components/Predefined/Details/VirtualServices/TlsRoutes.js

```diff
@@ -11,11 +11,15 @@ function TlsMatchAttributesItem({ match }) {
   return (
     <dl>
       <dd>{t('virtualservices.matches.sni-hosts')}</dd>
-      <dt>{match.sniHosts.join(', ')}</dt>
+      {match.sniHosts.map(host => (
```

Comment:
```suggestion
      {match.sniHosts?.map(host => (
```
Better safe than sorry

Reply:
technically it's required...